content/blog/2025-05-13-bril-concurrency-extension.md

```diff
@@ -0,0 +1,163 @@
++++
+title = "Sailing Bril’s IR into Concurrent Waters"
+[extra]
+bio = """
+  Ananya Goenka is an undergraduate studying CS at Cornell. While she isn’t nerding out over programming languages or weird ISA quirks, she’s writing for Creme de Cornell or getting way too invested in obscure books.
```

Comment:
[The magazine](https://www.cremedecornell.net) looks cool! Would you be interested in adding a hyperlink? :smiley: